WikipediaUnitTests/Code/TWNStringsTests.m

```diff
@@ -209,7 +208,9 @@ - (void)testIncomingTranslationStringForBracketSubstitutions {
     }
 }
 
-- (void)testiOSTranslationStringForBracketSubstitutions {
+- (void)testiOSTranslationStringForBracketSubstitutionsAndMismatchedTokens {
```

Comment:
Thoughts on maybe keeping this as two separate tests and possibly a utility method for any common bits?